rgf/utils.py

```diff
@@ -441,20 +413,27 @@ def cleanup(self):
         n_removed_files : int
             Returns the number of removed files.
         """
+        n_removed_files = 0
+        if self._estimators is not None:
+            for est in self._estimators:
+                n_removed_files += cleanup_partial(est._file_prefix,
+                                                   remove_from_list=True)
+
         # No more able to predict without refitting.
         self._fitted = None
-        return cleanup_partial(self._file_prefix, remove_from_list=True)
+        return n_removed_files
 
+    def _get_params(self):
+        raise NotImplementedError(NOT_IMPLEMENTED_ERROR_DESC)
+
+    def _fit_binary_task(self, X, y, sample_weight, params):
+        raise NotImplementedError(NOT_IMPLEMENTED_ERROR_DESC)
+
+    def _fit_multiclass_task(self, X, y, sample_weight, params):
+        raise NotImplementedError(NOT_IMPLEMENTED_ERROR_DESC)
 
-class RGFClassifierBase(RGFMixin, BaseEstimator, ClassifierMixin):
-    @property
-    def estimators_(self):
-        """The collection of fitted sub-estimators when `fit` is performed."""
-        if self._estimators is None:
-            raise NotFittedError(NOT_FITTED_ERROR_DESC)
-        else:
-            return self._estimators
 
+class RGFClassifierMixin(object):
```

Comment:
Add `_get_sample_weight()` and `_check_n_features()` stubs with `NotImplementedError` to `RGFClassifierMixin`